doc/nrf/device_guides/working_with_nrf/nrf70/gs.rst

```diff
@@ -1,26 +1,27 @@
 .. _nrf7002dk_nrf5340:
+.. _ug_nrf7002_gs:
 
-Getting started with nRF70 Series
-#################################
+Getting started with nRF7002 DK
+###############################
 
 .. contents::
    :local:
    :depth: 4
 
-This page gets you started with your nRF70 Series devices using the |NCS|.
+This page gets you started with your nRF7002 :term:`Development Kit (DK)` using the |NCS|.
+It tells you how to install the :ref:`wifi_scan_sample` sample and perform a quick test of your DK.
 
-If you want to go through an online training course to familiarize yourself with Wi-Fi® and the development of Wi-Fi applications, enroll in the `Wi-Fi Fundamentals course`_ in the `Nordic Developer Academy`_.
-
-Supported development boards
-****************************
+If you have already set up your nRF7002 DK and want to learn more, see the following documentation:
 
-nRF7002 DK
-==========
+* :ref:`installation` and :ref:`configuration_and_build` documentation to install the |NCS| and learn more about its development environment.
+* :ref:`ug_nrf70_developing` documentation for more advanced topics related to the nRF70 Series.
```

Comment:
Just a note - i think this part will not exist once the Protocols' rework PR is merged.

Reply:
Or at least, it won't include all the "Developing With" content so we need to have cross-links to all content (that will be split in two places)

Reply:
Added link to Protocols page as well.